### ISSUE TITLE
Meta transactions

### ISSUE THREAD

**Issue by DennisVonDerBey** [veramolabs/did-eth#9]
We should move away from the proprietary way of doing meta transactions in 'did:ethr' by using a public standard.

[EIP-712](https://eips.ethereum.org/EIPS/eip-712) would allow a good human readable way of doing that. I see value in doing that human readability especially if we consider a dApp to manage your DID in the future (think ens dApp) with the extended feature of letting some entity do a direct FIAT conversion feature by leveraging meta transactions ("let our payment wallet handle your transaction and pay us 5$"). Then a user could finally really see what their wallet is signing to cross check it.

Is there some alternative to EIP-712?

**Comment by strumswell** [comment]
I totally agree with you. What we should keep in mind here is the weird state EIP-712 is in currently due to its specification not being fully reflected in its implementations.

https://github.com/ethereum/EIPs/pull/5457
